### DISCUSSION TITLE
图片上传限制128kb || image upload limit 128kb

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2047] [Q&A] [answered]
### Describe the bug | 问题描述

我已看过 #1727的内容，但是修改default.ts后仍然有限制
![image](https://github.com/walinejs/waline/assets/90504717/a1e7183e-1c70-4888-8eba-b860ac999985)

### Website URL | 问题网站

https://2176608.github.io

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

I've seen #1727, but there are still limitations after modifying default.ts
![image](https://github.com/walinejs/waline/assets/90504717/a1e7183e-1c70-4888-8eba-b860ac999985)

### Website URL | Problem Website

https://2176608.github.io

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is your comment data store?

GitHub

**Comment by lizheming** [comment, marked as answer]
按照文档自定义，不需要改项目源码 https://waline.js.org/cookbook/customize/upload-image.html
